Uno.Gallery/Uno.Gallery.UITest/Constants.cs

```diff
@@ -14,6 +14,6 @@ public class Constants
         public readonly static string AndroidAppName = "com.nventive.uno.ui.demo";
         public readonly static string iOSDeviceNameOrId = "iPad Pro (12.9-inch) (4th generation)";
 
-        public readonly static Platform CurrentPlatform = Platform.Android;
+        public readonly static Platform CurrentPlatform = Platform.Browser;
```

Comment:
keep this change locally when needed to change the platform but not needed for this PR